keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+[_NUMPAD] = LAYOUT_ortho_4x12(
+  _______, _______, _______, _______, _______, _______,       KC_7,    KC_8,   KC_9,    _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       KC_4,    KC_5,   KC_6,    _______, _______, _______,
+  VAI_NEE, VAD_NEE, SAI_NEE, SAD_NEE, HUI_NEE, HUD_NEE,       KC_1,    KC_2,   KC_3,    _______, _______, _______,
+  _______, _______, _______, RGB_PRE, RGB_ANM, TOG_NEE,       KC_CALC, KC_0,   KC_PPLS, _______, _______, _______
+),
+
+/* Gaming
+ * ,-----------------------------------------.               ,-----------------------------------------.
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      | RAISE|      |               |      | LOWER|      |      |      |      |
+ * `-----------------------------------------'               `-----------------------------------------'
+ */
+[_GAMING] = LAYOUT_ortho_4x12(
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, RAISE,   _______,       _______, LOWER,   _______, _______, _______, _______
+)};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_BREATHING + 1;
+	autoshift_enable();
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case ADJUST:
+      if (record->event.pressed) {
+        layer_on(_ADJUST);
+      } else {
+        layer_off(_ADJUST);
+      }
+      return false;
+      break;
```

Comment:
Since you're already using `layer_state_set_user`, you could remove these, and add the following after the enums:
```c
#define LOWER MO(_LOWER)
#define RAISE MO(_RAISE)
```
And then use https://docs.qmk.fm/#/ref_functions?id=update_tri_layer_statestate-x-y-z below.

Reply:
Because `MO` in `layer_state_set_user` function the things are working but not the way that I would like them to. 
For example: 
```    case _RAISE:
      rgblight_setrgb_at(255, 255, 255, RGBLED_NUM / 2);
      rgblight_mode_noeeprom(RGBLIGHT_MODE_STATIC_LIGHT);
      break;
```
I expect that one led changes to white while I'm pushing  RAISE and when I release it changes back, but right now It flashes to white for less then a sec and immediately changes back to the default case.
What did I miss here?